fish-be/src/main/game/player.ts

```diff
@@ -38,17 +37,39 @@ export class Player implements IPlayer {
     return false;
   }
 
-  removeCard(card: Card): boolean {
+  stealCardFrom(player: Player, card: Card): boolean {
```

Comment:
This function is doing more than steal card from a player. It's checking to see if the current player can steal from the half suit, check if the player's card exists, then proceed to remove and add the card.

Why are we doing so much logic in the player model? Could we move that outside into the controller instead? I think that this could be just 

```
      const card1 = player.removeCard(card);
      this.addCard(card1);
```